static/index.css

```diff
@@ -0,0 +1,77 @@
+html, body {
+    height: 100%;
+}
+body {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    padding: 20px;
+    background-color: #2c1e37;
+    box-shadow:
+        rgba(0, 0, 0, 0.137255) 0px 0px 4px 0px,
+        rgba(0, 0, 0, 0.278431) 0px 4px 8px 0px;
+    text-align: right;
+}
+
+#importExportBox {
+    padding-right: 20px;
+}
+#importButton::before {
+    content: 'Import';
+}
+.importing #importButton::before {
+    content: 'Cancel Import';
+}
+input[type=file] {
+    display: none;
+    color: #fff;
```

Comment:
This doesn't seem to have any effect on the file picker controls (there seem to be a bunch of hacky ways to fix this with overlaying elements, none of which seem particularly elegant).